modules/ding_library/ding_library.views_default.inc

```diff
@@ -213,6 +257,16 @@ function ding_library_views_default_views() {
   $handler->display->display_options['allow']['title_override'] = 'title_override';
   $handler->display->display_options['allow']['exposed_form'] = 0;
   $handler->display->display_options['allow']['fields_override'] = 0;
+  $handler->display->display_options['argument_input'] = array(
```

Comment:
Looks like the autocomplete for events in search field? If yes, should not be ported.

Reply:
Not quite. This is the filter value taken from the current path.
This view being inserted on a certain page, will take the term id from the page's url to filter libraries within this term.
The page itself is bound to `libraries/!tid` path, so this !tid argument is fed to the view too.

Reply:
I don't get it. Can you show this to me visually?

Reply:
`admin/structure/views/view/ding_library/edit`
![image](https://user-images.githubusercontent.com/574498/100219256-6d0c9e80-2f1e-11eb-8313-681a8cd29129.png)

Reply:
This would be the result:
![image](https://user-images.githubusercontent.com/574498/100754179-6cc74400-33f3-11eb-9a78-93dc94d58ef7.png)
![image](https://user-images.githubusercontent.com/574498/100754218-794b9c80-33f3-11eb-804b-ad5e02b2172e.png)
However, such result requires changes the layout of libraries page. Those changes lack in this PR, as they deliver conflicts in features export files.

Reply:
You know what, this Library Type thing is great - but it changes too much. Please move everything related to this, to a separate module.